app/models/user.rb

```diff
@@ -87,7 +87,7 @@ def has_no_credentials?
   has_many :offer_assignments, :through => :pseuds
   has_many :pinch_hit_assignments, :through => :pseuds
   has_many :request_claims, :class_name => "ChallengeClaim", :foreign_key => 'claiming_user_id', :inverse_of => :claiming_user
-  has_many :gifts, :through => :pseuds
+  has_many :gifts, :through => :pseuds, :conditions => {rejected: false}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.